classes/autoptimizeCriticalCSSBase.php

```diff
@@ -20,29 +20,38 @@ public function __construct()
     {
         // define constant, but only once.
         if ( ! defined( 'AO_CCSS_DIR' ) ) {
-            // Define plugin version.
-            define( 'AO_CCSS_VER', 'AO_' . AUTOPTIMIZE_PLUGIN_VERSION );
-
             // Define a constant with the directory to store critical CSS in.
             if ( is_multisite() ) {
                 $blog_id = get_current_blog_id();
                 define( 'AO_CCSS_DIR', WP_CONTENT_DIR . '/uploads/ao_ccss/' . $blog_id . '/' );
             } else {
                 define( 'AO_CCSS_DIR', WP_CONTENT_DIR . '/uploads/ao_ccss/' );
             }
-
-            // Define support files locations.
-            define( 'AO_CCSS_LOCK', AO_CCSS_DIR . 'queue.lock' );
-            define( 'AO_CCSS_LOG', AO_CCSS_DIR . 'queuelog.html' );
-            define( 'AO_CCSS_DEBUG', AO_CCSS_DIR . 'queue.json' );
+        }
+        if ( ! defined( 'AO_CCSS_VER' ) ) {
+            // Define plugin version.
+            define( 'AO_CCSS_VER', 'AO_' . AUTOPTIMIZE_PLUGIN_VERSION );
 
             // Define constants for criticalcss.com base path and API endpoints.
             // fixme: AO_CCSS_URL should be read from the autoptimize availability json stored as option.
             define( 'AO_CCSS_URL', 'https://criticalcss.com' );
             define( 'AO_CCSS_API', AO_CCSS_URL . '/api/premium/' );
+        }
+        if ( ! defined( 'AO_CCSS_SLEEP' ) ) {
```

Comment:
Almost there; can you move this one back in the conditional block? this is not meant to be change-able by users, settings is agreed with the criticalcss.com team :-)